Makefile

```diff
@@ -0,0 +1,72 @@
+# opentrons modules makefile
+# https://github.com/Opentrons/opentrons-modules
+
+SHELL := /bin/bash
+
+LIBRARIES_DIR := libraries
+MODULES_DIR := modules
+INO_DIR := $(HOME)/arduino_ide
+
+ARDUINO_VERSION ?= 1.8.5
+CI_SYSTEM ?= linux
+
+ifeq ($(CI_SYSTEM), osx)
+	ARDUINO := $(INO_DIR)/Arduino.app/Contents/MacOS/Arduino
+	ARDUINO_ARCHIVE := arduino-$(ARDUINO_VERSION)-macosx.zip
+	UNPACK_CMD := unzip $(ARDUINO_ARCHIVE) -d $(HOME)/arduino_ide/
+	ARDUINO_SKETCHBOOK_LOC := $(HOME)/Documents/Arduino
+	ARDUINO15_LOC := $(HOME)/Library/Arduino15
+else
+	ARDUINO := $(INO_DIR)/arduino
+	ARDUINO_ARCHIVE := arduino-$(ARDUINO_VERSION)-linux64.tar.xz
+	UNPACK_CMD := tar xf $(ARDUINO_ARCHIVE) -C $(HOME)/arduino_ide/ --strip-components=1
+	ARDUINO_SKETCHBOOK_LOC := $(HOME)/Arduino
+	ARDUINO15_LOC := $(HOME)/.arduino15
+endif
+
+BUILD_PATH := $(HOME)/.arduino_builds
+DOWNLOAD_LINK := https://downloads.arduino.cc/$(ARDUINO_ARCHIVE)
+
+ifeq ($(wildcard $(ARDUINO)), )
+	NO_ARDUINO := true
+endif
+
+# setup Arduino
+.PHONY: setup-ino
+setup-ino:
+	# Download Arduino
+	mkdir -p $(HOME)/arduino_ide
+	# if not already cached, download and install/unpack arduino IDE
+	$(if $(NO_ARDUINO), wget --quiet $(DOWNLOAD_LINK), @echo "Arduino already present")
```

Comment:
`wget` is not present by default on macOS; can this script use `curl -O` instead?